apps/daimo-mobile/src/view/TabNav.tsx

```diff
@@ -82,6 +86,49 @@ export function TabNav() {
 
   if (!isOnboarded) return <OnboardingScreen {...{ onOnboardingComplete }} />;
 
+  // Error modal slides up from the bottom, greying out the app below.
```

Comment:
I understand what this is doing now!

So it basically recreates the iOS native "fade into background" effect for whatever is behind the modal

Looks clean

Lets ship it